### PR TITLE
Added private and public icons to the index, Also fixed Delete buttons

### PR DESCRIPTION
**Pull request by AlexW555**
A pretty small ticket as public and private was already done by Gareth in the create project ticket.

But added little icons for logged-in users who are project owners. This allows them to see what projects of theirs are public/private. ✅

Also changed the delete project link  into a button ✅

Screenshot 2021-08-17 at 3 49 17 pm

Screenshot 2021-08-17 at 3 49 26 pm

### CODE REVIEW

**Comment by phil-6** [comment]
I think this would look a little bit better with a bit of extra padding around the public/private icon. Maybe make it so its evenly spaced.

Do we want to show the likes button on private comments?

**Comment by garethdandrews** [comment]
I think we should add an 'actions' section, that is fixed to the bottom right of the card that contains the public/private status and then later, an edit icon.

File: app/views/projects/_project_card.html.erb [hunk 1]
```diff
@@ -52,4 +52,17 @@
       </div>
     </div>
   </div>
-</div>
\ No newline at end of file
+</div>
+
+
+<%
+=begin%>
```
Comment (IeuanSkinner) [inline app/views/projects/_project_card.html.erb:59]: What are these =begin/end tags for?

File: app/views/projects/show.html.erb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <section class="projects show">
   <%= render 'project_card', locals: { project: @project } %>
   <% if @project.user == current_user %>
-    <%= link_to project_path, 'Delete', method: :delete %>
+    <%= button_to 'Delete This Project', project_path, method: :delete, class: 'btn btn-danger' %>
```
Comment (IeuanSkinner) [inline app/views/projects/show.html.erb:4]: Hesistancy check should be added to major actions like deleting a project, look into the data-confirm functionality we use in lots of other Rails projects.

File: app/views/projects/show.html.erb [hunk 3]
```diff
@@ -29,4 +29,4 @@
       <% end %>
     </div>
   </section>
-</main>
\ No newline at end of file
+</section>
```
Comment (IeuanSkinner) [inline app/views/projects/show.html.erb:32]: This isn't right, section should not be nested within a section.

File: app/views/projects/_project_card.html.erb (hunk 1 above)
Comment (garethdandrews) [reply app/views/projects/_project_card.html.erb]: I pushed a change to the branch whilst I'm working on something.

**Comment by garethdandrews** [comment]
I have slightly restyled the project card on member view. If the user is the project owner, they will now see an 'action bar', with a public/private status, and an edit button. The delete button has been moved to the edit project form. Additionally, a confirmation check has been added before deleting from the database. 

Screenshot 2021-08-18 at 15 48 31

Screenshot 2021-08-18 at 15 49 09

@IeuanSkinner

**Comment by IeuanSkinner** [comment]
@garethdandrews because they sit above the card itself it's not readily clear that the public/private status refers to that project or that the edit button does either. Could they be bought into the card area?

**Comment by garethdandrews** [comment]
@IeuanSkinner no problem

Screenshot 2021-08-19 at 09 24 07
Screenshot 2021-08-19 at 09 24 15

**Comment by garethdandrews** [comment]
Whilst I was on this branch I also added the functionality to hide projects that aren't public from the main index. They are however visible on the users profile, just greyed out.

Screenshot 2021-08-19 at 09 47 43

**Comment by phil-6** [comment]
Loving the improvements here, I would like to see an icon to incident why a project is greyed out.

**Comment by garethdandrews** [comment]
@phil-6 changes made 👍 

Screenshot 2021-08-19 at 11 32 21

**Pull request merged**